Modules/ELCodable/ELCodable/Decimal.swift

```diff
@@ -66,42 +66,42 @@ extension Decimal /*: FloatingPointType*/ { // It complains about _BitsType miss
     }
     /// `true` iff `self` is negative.
     public var isSignMinus: Bool {
-        return ((value as Double).sign == .minus)
```

Comment:
This entire file should move to use the Swift version of Decimal, thus becoming ...

```
extension Decimal {
    public var value: NSDecimalNumber {
        get {
            return (self as NSDecimalNumber)
        }
    }
}
```

and that's it.